rosdep/python.yaml

```diff
@@ -6309,6 +6309,7 @@ wxpython:
     artful: [python-wxgtk3.0]
     bionic: [python-wxgtk3.0]
     cosmic: [python-wxgtk3.0]
+    focal: [python-wxgtk3.0]
```

Comment:
For all of our currently-supported Ubuntu releases, the key is `python-wxgtk3.0`.  So I think we should just replace this whole block with:

```
ubuntu: [python-wxgtk3.0]
```

Which has the added benefit of making us forwards-compatible.

Reply:
@clalancette thanks! chaned to keep both backward/forward compatible